pkg/util/status/status_test.go

```diff
@@ -266,7 +266,7 @@ var _ = Describe("Status", func() {
 			By("checking the VirtualMachineInstanceMigration resource")
 			migrationUpdater := NewMigrationStatusUpdater(virtClient)
 			migration := &v1.VirtualMachineInstanceMigration{Status: v1.VirtualMachineInstanceMigrationStatus{Phase: v1.MigrationPhaseUnset}}
-			migrationInterface.EXPECT().UpdateStatus(migration).Return(migration, nil).Times(1)
+			migrationInterface.EXPECT().UpdateStatus(context.Background(), migration, v12.UpdateOptions{}).Return(migration, nil).Times(1)
```

Comment:
nit: There is `metav1`

Reply:
nice catch! I removed the duplicate import in te `UpdateStatus` commit